data/battle_scripts_1.s

```diff
@@ -2128,9 +2128,15 @@ BattleScript_EffectHitSwitchTarget:
 	moveendall
 	jumpifability BS_TARGET, ABILITY_SUCTION_CUPS, BattleScript_AbilityPreventsPhasingOut
 	jumpifstatus3 BS_TARGET, STATUS3_ROOTED, BattleScript_PrintMonIsRooted
-	tryhitswitchtarget BattleScript_EffectHitSwitchTargetMoveEnd
-BattleScript_EffectHitSwitchTargetMoveEnd:
-	end
+	tryhitswitchtarget BattleScript_MoveEnd
+
```

Comment:
`tryhitswitchtarget` either jumps to `failInstr` or directly to `BattleScript_HitSwitchTargetForceRandomSwitch`. Originally I formatted it like you did, but then I thought this formatting was a little better, because somebody might make this change:
```diff
tryhitswitchtarget BattleScript_MoveEnd
+command here
BattleScript_HitSwitchTargetForceRandomSwitch::
	forcerandomswitch BattleScript_HitSwitchTargetForceRandomSwitchFailed
```
Which wouldn't work!

Maybe `tryhitswitchtarget` should just set `gBattleScripting.switchCase = B_SWITCH_HIT` and then fall through?

Reply:
Yeah, this seems good